assets/src/components/parts/types/parts.ts

```diff
@@ -70,3 +70,13 @@ export interface ColorPalette {
   lineStyle?: number;
   lineThickness?: number;
 }
+
+export const defaultCapabilities = {
+  move: true,
```

Comment:
will `move` also encompass `z` as well as `x` & `y` ?

Reply:
yes, I was thinking so, and have enabled both drag and the fwd/back buttons based on it